### ISSUE TITLE
[1.21] Duplicated tags on items

### ISSUE THREAD

**Issue by ffuentesm** [Sinytra/ForgifiedFabricAPI#178]
The mod is using outdated Tags and causing a duplicated entry. The correct tag id is "c:foods/vegetable" (not plural).
https://github.com/Sinytra/ForgifiedFabricAPI/blob/1.21.1/fabric-convention-tags-v2/src/generated/resources/data/c/tags/item/foods/vegetables.json

Carrots, Potatoes, beets, and Golden Carrots all have duplicated tags c:foods/vegetable and c:foods/vegetables
![Image](https://github.com/user-attachments/assets/3b4e589c-a767-477b-80f8-8555dc32c67c)
![Image](https://github.com/user-attachments/assets/7a28de75-2008-457a-956c-e646d9df8f95)
![Image](https://github.com/user-attachments/assets/9b5ae7a7-069a-43b6-9a52-788d13d40f7f)
![Image](https://github.com/user-attachments/assets/8dda40c5-e49d-4945-b60b-79cc7107b3d4)

These are not just the only ones. Many other tags have changed and it's causing the duplicated tags. 
https://github.com/neoforged/NeoForge/tree/1.21.x/src/generated/resources/data/c/tags/item

**Comment by Su5eD** [comment]
This seems to be consistent with upstream FAPI' [tags](https://github.com/FabricMC/fabric/tree/1.21.4/fabric-convention-tags-v2/src/generated/resources/data/c/tags/item/foods). Are there any in-game side effects caused by this that would require attention?

**Comment by ffuentesm** [comment]
Yes, adding c:coal to the #minecraft:coals tag is breaking some of my recipes
https://github.com/Sinytra/ForgifiedFabricAPI/blob/9c40919e2eefc42d6e2a7a04a823732151ef69a0/fabric-convention-tags-v2/src/generated/resources/data/c/tags/item/coal.json#L4

This is causing the c:coal tag to be added to charcoal and coal coke items from other mods I have. It breaks the recipes that use the c:charcoal and c:coal_coke tags. To fix that, you just have to unappend the c:coal from the #minecraft:coals tag and instead add the minecraft:coal item.
